pkg/kubectl/cmd/proxy.go

```diff
@@ -83,14 +83,33 @@ func NewCmdProxy(f *cmdutil.Factory, out io.Writer) *cobra.Command {
 	cmd.Flags().StringP("address", "", "127.0.0.1", "The IP address on which to serve on.")
 	cmd.Flags().Bool("disable-filter", false, "If true, disable request filtering in the proxy. This is dangerous, and can leave you vulnerable to XSRF attacks, when used with an accessible port.")
 	cmd.Flags().StringP("unix-socket", "u", "", "Unix socket on which to run the proxy.")
+	cmd.Flags().StringP("curl", "", "", "Raw URI to request from the proxy server.")
```

Comment:
Yes, my intention was that the proxy server would start, then the command would be executed passing in the proxy server url, then after the command exists the proxy server is torn down.  It may be worth considering using a random port for the proxy server so you could run multiple commands in parallel.